astropy/utils/data.py

```diff
@@ -1793,25 +1797,27 @@ def import_download_cache(filename_or_obj, urls=None, update_cache=False):
     See Also
     --------
     export_download_cache : export the contents the cache to of such a ZIP file
-    import_to_cache : import a single file directly
+    import_file_to_cache : import a single file directly
     """
-    block_size = 65536
     with zipfile.ZipFile(filename_or_obj, 'r') as z, TemporaryDirectory() as d:
         for i, zf in enumerate(z.infolist()):
             url = urllib.parse.unquote(zf.filename)
+            # FIXME(aarchiba): do we want some kind of validation on this URL?
```

Comment:
As long as users stick to using the provided function to create the ZIP file, validation is not strictly necessary, right? And if they are creating the ZIP by hand, then they are on their own.

But if you are talking about ZIP being corrupted on transport, then file checksum would be sufficient?

Reply:
I guess I'm more thinking the user could be surprised if they end up with a bunch of strange non-URLs as keys in their cache. But they can supply any string at all as `remote_url` as long as `sources` points somewhere useful or they're using `import_file_to_cache`. So this doesn't really introduce new headaches. I was thinking more of problems with the quote/unquote operations.

Reply:
Ah, okay. Thanks for the clarification!